IsraelHiking.Web/src/application/services/route-statistics.service.spec.ts

```diff
@@ -195,25 +195,25 @@ describe("RouteStatisticsService", () => {
 
         let statistics = service.getStatistics(routeData, null, null, null, false);
 
-        expect(statistics.gain).toBeCloseTo(59.9,1);
-        expect(statistics.loss).toBeCloseTo(-59.8,1);
+        expect(statistics.gain).toBeCloseTo(278.4,1);
+        expect(statistics.loss).toBeCloseTo(-238.4,1);
```

Comment:
This is strange, isn't it? the `700` shouldn't affect the calculations, shouldn't it?
In previous results it was "wiped out" by the median filter and now it seems that it has some effect, doesn't it?